src/pt/randomscan/build.gradle

```diff
@@ -3,7 +3,7 @@ ext {
     extClass = '.LuraToon'
     themePkg = 'peachscan'
     baseUrl = 'https://luratoon.com'
-    overrideVersionCode = 39
+    overrideVersionCode = 40
```

Comment:
Should bump PeachScan instead. Couldn't the `imageRequest` function also be moved there?